index-scheduler/src/snapshots/lib.rs/cancel_enqueued_task/cancel_processed.snap

```diff
@@ -1,6 +1,5 @@
 ---
 source: index-scheduler/src/lib.rs
-assertion_line: 1755
```

Comment:
I don't really have any idea why there was an `assertion_line` here and what it means and why it is not here anymore.

Reply:
Don't know either but if that makes insta happy I'm 100% for it 😂